README.md

```diff
@@ -12,3 +12,53 @@ Grafana or the like.
 This is a [Haskell stack tools](https://docs.haskellstack.org/) based
 project.  Given a working `stack`, one should be able to execute
 `stack build` which will fetch all of the dependencies and build.
+
+After building, you will have several executables.
+
+teslauth: Authenticates to Tesla's API (with your credentials) and
+stores them in a local database.  _For obvious reasons_ treat this
+carefully.
+
+teslacatcher: Drains an MQTT topic to a sqlite database.
```

Comment:
I don't want to overwhelm the user with documentation, but this can also backfill missing data, effectively synchronizing state locally.  Perhaps that's what the wiki is for.

Reply:
That's not too onerous here.